src/styles/_normolize.scss

```diff
@@ -0,0 +1,20 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
```

Comment:
why html and body located in normilize file?
I think better will be  move this styles to common.scss